app/views/wrapped/show.html.haml

```diff
@@ -0,0 +1,62 @@
+- model_class = User
+.page-header
+  %h1
+    = "#{@user.name}'s #{@year} wrapped"
+    %small
+      the last twelve months
+
+.alert.alert-info
+  This is a new feature. Be sure to leave feedback on
+  = link_to "GitHub", "https://github.com/chaosdorf/mete/pull/108"
```

Comment:
this is a repetition of app/views/wrapped/index.html.haml#L9-10 - consider using a view partial